scopes/component/ui/compare/component-compare.tsx

```diff
@@ -55,13 +57,17 @@ export function ComponentCompare({ navSlot, host, routeSlot }: ComponentCompareP
 
   const { component: base, loading } = useComponent(host, baseId.toString());
 
+  const nothingToCompare = !compareIsLocalChanges && (component.logs?.length || []) < 2;
```

Comment:
I would add `!loading` here as well. otherwise the loader and empty compare can show at the same time